src/autoscaler/operator/config/config.go

```diff
@@ -109,6 +120,9 @@ var defaultConfig = Config{
 		LockRetryInterval: DefaultRetryInterval,
 		LockTTL:           DefaultLockTTL,
 	},
+	AppSyncer: AppSyncerConfig{
+		SyncInterval: DefaultSyncInterval,
```

Comment:
the DefaultSyncInterval (10 minutes) is too short. Imagine we may have 10K apps,  it will take a pretty long time.  And I think remove the orphan app policies is not that urgent so we probably can give it one day as default sync interval